PELE-466; Frag Refactoring (#131)
nostrumbiodiscovery/pele_platform: docs/build_docs/source/packages/libraries/index.rst

    frag_core_atom: "C6-H6"  # Connection point from which you want to grow the fragments
    cpus: 48

Optional Parameters
+++++++++++++++++++++++

> * **frag_restart**: To resume a fragment libraries simulation.

Reviewer: This is now outdated, right?

Author: Yes. Changing it now.

> * **fragment_atom**: To select an hydrogen of the fragment which we want to attach it to the **frag_core_atom**, instead of performing a simulation from every hydrogen on the fragment. 

Reviewer 2: ```suggestion
* **fragment_atom**: Select a hydrogen of the fragment that we want to attach to the **frag_core_atom**, instead of performing a simulation from every hydrogen of the fragment. 
```
